site/docs/uk/hosting/vps.md

```diff
@@ -52,130 +152,631 @@ systemd --- це потужний менеджер служб, який пере
+```
+
+:::
+
+### Оренда домену
+
+Щоб звʼязати бота, який працює на вебхуках, із зовнішнім світом, потрібно придбати домен.
+
+Перейдіть на [сторінку пошуку доменних імен](https://hostinger.com.ua/kupyty-domen).
+У полі введення тексту введіть доменне імʼя виду `<назва>.<зона>`.
+Наприклад, `example.com`.
+
+Якщо бажаний домен вільний, натисніть кнопку "Додати" поряд із ним.
+Вас буде автоматично перенаправлено на сторінку оформлення замовлення, де ви також одразу зареєструєтеся у Hostinger, якщо досі не зареєстровані.
+Оплатіть домен.
+
+Щоб підключити бота, який працює на вебхуках, до зовнішнього світу, вам потрібно придбати домен.
+Ми знову будемо пояснювати це на прикладі Hostinger, але є багато інших сервісів, і всі вони працюють аналогічно.
+
+Перейдіть на [сторінку пошуку доменного імені](https://hostinger.com.ua/kupyty-domen).
+У полі для введення тексту введіть доменне ім'я у вигляді `<ім'я>.<зона>`.
+Наприклад, `example.com`.
+
+Якщо потрібний домен вільний, натисніть кнопку "Додати" поруч з ним.
+Ви будете автоматично перенаправлені на сторінку оформлення замовлення, де також відразу ж зареєструєтесь у Hostinger, якщо ви ще не зареєстровані.
+Оплатіть домен.
+
+#### Направлення домена на VPS
+
+Перш ніж ваш домен зможе працювати з вашим VPS, вам потрібно направити домен на ваш сервер.
+Для цього в [панелі керування Hostinger](https://hpanel.hostinger.com) натисніть кнопку "Керування" навпроти вашого домену.
+Далі перейдіть на сторінку управління DNS-записами, натиснувши на кнопку "DNS / Сервери імен" в меню ліворуч.
+
+> Спершу дізнайтеся IP-адресу свого VPS.
+
+У списку DNS-записів знайдіть запис типу `A` з імʼям `@`.
+Відредагуйте цей запис, змінивши IP-адресу в полі "Вказує на" на IP-адресу вашого VPS і встановивши TTL рівним 3600.
+
+Далі знайдіть і видаліть запис типу `CNAME` з імʼям `www`.
+Замість нього створіть новий запис типу `A` з імʼям `www`, що вказуватиме на IP-адресу вашого VPS, і встановіть TTL рівним 3600.
+
+> Якщо у вас виникнуть проблеми, скористайтеся іншим способом, описаним в [базі знань](https://support.hostinger.com/uk/articles/1583227-%D1%8F%D0%BA-%D0%BD%D0%B0%D0%BF%D1%80%D0%B0%D0%B2%D0%B8%D1%82%D0%B8-%D0%B4%D0%BE%D0%BC%D0%B5%D0%BD-%D0%BD%D0%B0-vps).
+
+### Налаштування вебсервера
+
+Щоб сайт запрацював, а бот почав отримувати оновлення від Telegram, потрібно налаштувати вебсервер.
+Ми скористаємося [Caddy](https://caddyserver.com).
+
+Caddy --- це потужний вебсервер з відкритим вихідним кодом і автоматичним HTTPS.
+
+::: tip Вебсервер
+Ми використовуємо Caddy, тому що, на відміну від поширених вебсерверів, як-от Nginx або Apache, він автоматично налаштовує SSL-сертифікати.
+Це значно спрощує посібник.
+Втім, ви можете вибрати будь-який вебсервер, який вам подобається.
+:::
+
+#### Встановлення
+
+Наступні 5 команд завантажать й автоматично запустять Caddy як службу systemd з назвою `caddy`.
 
 ```sh
-systemctl stop <назва-служби>
+apt install -y debian-keyring debian-archive-keyring apt-transport-https curl
```

Comment:
```suggestion
apt update
apt install -y debian-keyring debian-archive-keyring apt-transport-https curl
```

Reply:
This is a sample code from the official documentation. I think they know what they are doing.

Reply:
In the examples, we use `apt` and `apt-get`. There's really not much difference, but it would be cool if we kept the same type of command throughout the text.

Reply:
As far as I know now, apt is the newer and better version of apt-get. So what are the advantages of apt-get over apt? The installation commands are the same.

As for the fact that we've already used apt-get somewhere, I don't think it's a big problem. There is already a lot of heterogeneity in the documentation, but that doesn't bother us.

Reply:
TLDR: `apt` is for the end user, `apt-get` is for scripts.

> SCRIPT USAGE AND DIFFERENCES FROM OTHER APT TOOLS

> The [apt(8)](https://manpages.debian.org/bookworm/apt/apt.8.en.html) commandline is designed as an end-user tool and it may change behavior between versions. While it tries not to break backward compatibility this is not guaranteed either if a change seems beneficial for interactive use.

> All features of [apt(8)](https://manpages.debian.org/bookworm/apt/apt.8.en.html) are available in dedicated APT tools like [apt-get(8)](https://manpages.debian.org/bookworm/apt/apt-get.8.en.html) and [apt-cache(8)](https://manpages.debian.org/bookworm/apt/apt-cache.8.en.html) as well. [apt(8)](https://manpages.debian.org/bookworm/apt/apt.8.en.html) just changes the default value of some options (see [apt.conf(5)](https://manpages.debian.org/bookworm/apt/apt.conf.5.en.html) and specifically the Binary scope). So you should prefer using these commands (potentially with some additional options enabled) in your scripts as they keep backward compatibility as much as possible.

Source: [manpages.debian.org](https://manpages.debian.org/bookworm/apt/apt.8.en.html#SCRIPT_USAGE_AND_DIFFERENCES_FROM_OTHER_APT_TOOLS)

Reply:
And that? Leave apt as in the official documentation?

Reply:
In that case, let's leave it as it is, we don't lose anything.